lib/ansible/plugins/connection/local.py

```diff
@@ -7,9 +7,9 @@
 
 DOCUMENTATION = '''
     name: local
-    short_description: execute on controller
+    short_description: execute on the Ansible control node
     description:
-        - This connection plugin allows ansible to execute tasks on the Ansible 'controller' instead of on a remote host.
+        - This connection plugin allows ansible to execute tasks on the Ansible control node instead of on a remote node.
```

Comment:
`remote` or `target` here ?

Reply:
This looks like it should remain remote.  Maybe `remote host` is the right thing here as well, though, not `remote node`.  That way both words remain different than what we're standardizing on for the `target node`

Reply:
```suggestion
        - This connection plugin allows Ansible to execute tasks on the Ansible control node instead of on a remote host.
```

Reply:
The reason it should be remote host is that, almost by definition, any connection plugin allows Ansible to execute tasks on the `target node`.  What's different about the local connection plugin is that instead of primarily being used to make a remote host the target, the local connection plugin can only target the `control node` to run the tasks on.